### ISSUE TITLE
About file

### ISSUE THREAD

**Issue by flipside17** [sheng-n/GCLMTP#1]
I find the file named 'lnc_mi_interaction' in datasets2 has some wrong. Can you check it and send me the correct version? my email is 395187997@qq.com

**Comment by sheng-n** [comment]
您好，暂时没有发现dataset2中mi_lnc_intersection.csv存在问题。


| |
shengnanwangyi
|
|
***@***.***
|
---- Replied Message ----
| From | ***@***.***> |
| Date | 11/1/2023 20:18 |
| To | ***@***.***> |
| Cc | ***@***.***> |
| Subject | [sheng-n/GCLMTP] About file (Issue #1) |

I find the file named 'lnc_mi_interaction' in datasets2 has some wrong. Can you check it and send me the correct version? my email is ***@***.***

**Comment by Within-yao** [comment]
How do you separate the training set from the test set?
